### ISSUE TITLE
Angular tooltip should be hoverable for accessibility

### ISSUE THREAD

**Issue by juanjosegutierrez** [vmware/clarity#6574]
## Describe the bug

The angular tooltip element opens when you hover over the trigger element and closes when you leave. If you try to hover over the tooltip content, then the tooltip will close. According to WCAG 21, tooltip content needs to be hoverable too. Here is the documentation: https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html.

## How to reproduce

This issue can be seen on the clarity page for the angular tooltip in one of the tooltip examples: https://clarity.design/angular-components/tooltip/

Here is a screenshot: 
<img width="1787" alt="Screen Shot 2022-01-13 at 1 12 24 PM" src="https://user-images.githubusercontent.com/19143833/149410028-488b000f-e1b8-4707-a0f3-379836a0f6cf.png"> 

1.  Go to https://clarity.design/angular-components/tooltip/
3.  Scroll down to the tooltip demos like in the screenshot.
4.  Hover over one of the tooltip demo elements.
5.  Once the tooltip opens, attempt to hover over the tooltip content.
6.  See that it is not possible

## Expected behavior

User should be able to hover over tooltip content.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 12

**Device:**

- Type: MacBook
- OS: MacOS
- Browser chrome
- Version 95

## Additional notes

None.

**Comment by gnomeontherun** [comment]
@scrosiner Can you check if this is already a duplicate of what we have tracked internally? I believe so.

@juanjosegutierrez is based on testing through VMware?

**Comment by juanjosegutierrez** [comment]
@gnomeontherun yes this is based on VMware testing
